pkg/osvscanner/osvscanner.go

```diff
@@ -266,10 +274,15 @@ func exportDockerImage(r reporter.Reporter, dockerImageName string) (string, err
 		return "", err
 	}
 
-	r.Infof("Pulling docker image (%q)...\n", dockerImageName)
-	err = runCommandLogError(r, "docker", "pull", "-q", dockerImageName)
-	if err != nil {
-		return "", fmt.Errorf("failed to pull container image: %w", err)
+	// Check if image exists locally, if not, pull from the cloud.
+	cmd := exec.Command("docker", "images", "-q", dockerImageName)
```

Comment:
Can you use the runCommandLogError function for this?

Reply:
I thought about it, but it doesn't return error when image not exists though.

Reply:
Hmm ok, can you add a print statement here about what this is doing, e.g. Locating image locally..., then inside the if statement, image not found locally

Reply:
Want the user to know what we are doing so they are not surprised by the behaviour.